README.md

```diff
@@ -93,6 +93,28 @@ export default function Input() {
 }
 ```
 
+### Using the debounced setter
+
+In addition to getting the debounced value, you can destructure a second element which is a debounced setter function. This allows you to use `useDebounce` without needing a separate state variable for the input value:
+
+```javascript
+import React from 'react';
+import { useDebounce } from 'use-debounce';
+
+export default function Input() {
+  const [state, setState] = useDebounce('', 300);
+
+  return (
+    <div>
+      <input onChange={(e) => setState(e.target.value)} />
+      <p>Debounced value: {state}</p>
+    </div>
+  );
+}
```

Comment:
The same example exist right above the added section. 
I don't think repetition is needed here